public/app/features/plugins/plugin_loader.ts

```diff
@@ -28,12 +28,13 @@ SystemJS.addImportMap({ imports });
 
 const systemJSPrototype: SystemJSWithLoaderHooks = SystemJS.constructor.prototype;
 
-// This instructs SystemJS to load a plugin using fetch and eval if it returns a truthy value, otherwise it will load the plugin using a script tag.
+// This instructs SystemJS to load a assets using fetch and eval if it returns a truthy value, otherwise it will load the plugin using a script tag.
 // We only want to fetch and eval plugins that are hosted on a CDN or are Angular plugins.
 systemJSPrototype.shouldFetch = function (url) {
   const pluginInfo = getPluginFromCache(url);
+  const moduleTypesRegEx = /^[^#?]+\.(css|html|json|wasm)([?#].*)?$/;
```

Comment:
I was wondering if it would be safer to specify what we would like to load using script tags instead of specifying what we don't? E.g.:

```ts
const jsTypeRegEx = /^[^#?]+\.(js)([?#].*)?$/;

return !jsTypeRegEx.test(url) || ...
```

Reply:
I'm not sure that'll work. Under certain conditions (like CDN loading or the plugin is using angular) we need to load js files via fetch and eval.

Reply:
I think that could work by shuffling around the conditions (angular and CDN check comes first, file type last) - but it's not that important, the current blacklist should probably cover everything.

Reply:
Sorry, you're correct. Shuffling the conditions around can certainly work. I've updated it to use your suggestion. Thanks! 👍